src/axom/sidre/core/Group.hpp

```diff
@@ -896,14 +896,34 @@ class Group
    * the View is not copied. The new View object is associated with
    * the same data as the original.
    *
-   * If given Group pointer is null or Group already has a child Group with
-   * same name as given Group, method is a no-op.
+   * If given View pointer is null or Group already has a View with
+   * same name as given View, method is a no-op.
    *
-   * \return pointer to given argument Group object or nullptr if Group
-   * is not moved into this Group.
+   * \return pointer to given argument View object or nullptr if View
+   * is not copied into this Group.
    */
   View* copyView(View* view);
 
+  /*!
+   * \brief Create a deep copy of given View object and add it to this Group.
+   *
+   * Note that for Views that use sidre Buffers or external data, the
+   * deep copy performs a copy of the data described by the View
+   * into a new Buffer attached to the destination View. If the source View
+   * describes only part of a Buffer or part of an external array, due to
+   * strides and or offsets into the data, only the values seen by the 
+   * description will be copied into the new View. The new View will have
+   * a Buffer that is the exact size of the number of values that are copied,
+   * with an offset of zero and a stride of one.
+   *
+   * If given View pointer is null or Group already has a View with
+   * same name as given View, method is a no-op.
+   *
+   * \return pointer to given argument Group object or nullptr if View
+   * is not copied into this Group.
```

Comment:
Returns a pointer to a View object (same fix as lines 902 and 903).

That said, it seems odd that `copyView` and `copyGroup` and their `deepCopy` variants return the pointer to the object given.  I think they should return the pointer to the newly created Group or View.  Perhaps that's a deeper design discussion, though.

Reply:
That's an interesting point. Yes, we should discuss the implications of this choice.

Reply:
Looking at Group.cpp (new) lines 966, 999, 1406, 1456, it appears we always return the new copy.

Reply:
Yes, the returned objects are the new copied objects, so I need to correct this text.